ajax/libs/orgchart/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "orgchart",
+  "description": "Simple and direct organization chart(tree-like hierarchy) plugin based on pure DOM and jQuery.",
+  "filename": "js/jquery.orgchart.min.js",
+  "keywords": [
+    "jquery",
+    "plugin",
+    "organization",
+    "chart",
+    "orgchart",
+    "tree",
+    "tree-like",
+    "tree-view"
+  ],
+  "author": "dabeng <dabeng413@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dabeng/OrgChart.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dabeng/OrgChart.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "+(css|js)/jquery.orgchart*"
+        ]
+      },
+      {
+        "basePath": "",
```

Comment:
I think we don't need this config of line 37 ~ 41 because there is no `jquery.orgchart*` file in the root of the repo(the current version), right?

Reply:
There are file in root in oldest versions!

Reply:
If there are in old version, we need to manually add them because our config is only for the current file structure. Can you remove the part of config? And our team will add the old versions later. Thank you for your reminding.